code/modules/mob/animations.dm

```diff
@@ -327,9 +327,13 @@ note dizziness decrements automatically in the mob's Life() proc.
 
 /atom/movable/proc/do_putdown_animation(atom/target, mob/user)
 	spawn()
+		if (!istype(target))
```

Comment:
Want to point out, how this doesn’t fix the runtime itself, barely masking it.

Reply:
how do you figure that?

Reply:
How do I figure that simply “checking” whether passed argument isn’t null(istype()) does not solve the problem of argument ever becoming null? Shouldn’t the proc not even be called if there is no target? I say spawn has something to do with this.

Reply:
because there are a variety of situations where target can be null legitimately and that causes a runtime error

Saying "spawn has something to do with this" is a strangely accusatory way to say "the target atom might have been deleted last frame". That's what a safety check is for